packages/react/src/components/ListBoxBase/ListBoxBase.styles.ts

```diff
@@ -18,4 +18,24 @@ export const useStyles = css({
     overflow: 'visible',
     width: 'auto',
   },
+
+  '.manifest-listbox-no-results-container': {
+    py: '$small',
+  },
+
+  '.manifest-listbox-loading-container': {
+    '.manifest-spinner': {
+      height: 24,
+      width: 24,
+      color: '$text-secondary',
+    },
+
+    '.manifest-typography--caption': {
+      alignSelf: 'center',
+      color: '$text-secondary',
+      pl: '$x-small',
+    },
+
+    py: '$small',
```

Comment:
[nit] this "global" style to the no-results-container should sit at the top to visually indicate its "globalness"